command/genesis/genesis.go

```diff
@@ -107,7 +105,7 @@ func setFlags(cmd *cobra.Command) {
 		cmd.Flags().StringVar(
 			&params.validatorPrefixPath,
 			command.IBFTValidatorPrefixFlag,
-			"",
+			"test-chain-",
```

Comment:
Why is this the default value?

Reply:
fixed, created another param which should be done in first place